cost/azure/hybrid_use_benefit_sql/ahub_sql.pt

```diff
@@ -241,6 +241,8 @@ script "js_filtered_instances", type: "javascript" do
   result "result"
   code <<-EOS
   var result = [];
+  sql_virtualmachines = _.find(sql_virtualmachines, function(vm){ return vm.imageSku != "Developer" && vm.imageSku != "Express" });
```

Comment:
instead of using find, Can we not move this condition inside below for loop ?

Reply:
No we cant do that, because if the array only contains an element, and that element has to be shifted, the policy will fail